angular/projects/admin-nrpti/src/app/mines/mines-records-detail/mines-records-detail.component.html

```diff
@@ -6,11 +6,20 @@ <h1 id="name" class="m-0 truncate">{{ (record && record.recordName) || '-' }}</h
   </section>
 
   <section class="mb-3 pb-3 border-bottom">
-    <div class="row">
+    <div class="row justify-content-between">
       <div class="mb-2 col-12 col-sm-8">
         <h2 class="border-0 mb-0">Record Information</h2>
         <span class="grey-subtext">{{ lastEditedSubText }}</span>
       </div>
+      <div class="d-flex"></div>
```

Comment:
This seems odd, is there a better way to do this?

Reply:
was unnecessary - has been removed